lib/plaidRequest.js

```diff
@@ -28,6 +28,16 @@ var handleApiResponse = function(resolve, reject, err, res, $body, isMfa) {
 
   // success response (non mfa)
   } else if (res.statusCode === 200) {
+    // extract request id from header for binary data,
+    // i.e. mime type application/*
+    if (res.headers['plaid-request-id'] &&
```

Comment:
we should not rely on the truthiness construct in javascript here - can we explicitly check `!= null` ?

Reply:
You mean check for `undefined`, right?

Reply:
Whether it is `null` or `undefined`, line 35 is going to throw an exception. We can cover both cases with `!= null`.

I think I "happen to know" that its going to be undefined, but there is no harm in being cautious and defensive here.

Reply:
@byF `!= null` checks for both `null` and `undefined`. It should be fine if we just make this line:
```javascript
if (res.headers['plaid-request-id'] != null &&
```
@byF thanks again for all your help here.

Reply:
Thanks for the explanation guys! Commit away.